public/components/custom_panels/helpers/custom_input_modal.tsx

```diff
@@ -64,6 +69,7 @@ export const CustomInputModal = (props: CustomInputModalProps) => {
     helpText,
     optionalArg1,
     optionalArg2,
+    optionalArg3,
```

Comment:
might be better use `...optionalArgs` to support arbitrary number of args?

Reply:
changed 👍🏽